composer.json

```diff
@@ -39,8 +39,8 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.7.1",
         "mikey179/vfsStream": "1.1.0",
-        "phpunit/phpunit": "^5.7",
-        "matthiasnoback/symfony-dependency-injection-test": "~1.0",
+        "phpunit/phpunit": "~6.0",
+        "matthiasnoback/symfony-dependency-injection-test": "~2.0",
```

Comment:
Like the above comment, this should be `^1.0|^2.0` to support testing on both PHP 5.6 and PHP 7.x

Reply:
master does not support php 5.6 anymore ;)

Reply:
👍